packages/cli/test/functional/test_site_special_tags/expected/index.html

```diff
@@ -31,19 +32,19 @@ <h1 id="functional-test-for-htmlparser2-and-markdown-it-patches-for-special-tags
     <h2 id="so-far-as-to-comply-with-the-commonmark-spec"><span id="so-far-as-to-comply-with-the-commonmark-spec" class="anchor"></span>So far as to comply with the commonmark spec<a class="fa fa-anchor" href="#so-far-as-to-comply-with-the-commonmark-spec" onclick="event.stopPropagation()"></a></h2>
     <p>There should be no text between this and the next <code class="hljs inline no-lang" v-pre>&lt;hr&gt;</code> tag in the browser, since it is a <code class="hljs inline no-lang" v-pre>&lt;script&gt;</code> tag.<br>
       There should be an alert with the value of 2 as well.</p>
-    <script>
+    <script type="application/javascript">
       let x = 1;
 
       x += 1;
       alert(x);
     </script>
     <hr>
     <p>There should be no text between this and the next <code class="hljs inline no-lang" v-pre>&lt;hr&gt;</code> tag in the browser, since it is a <code class="hljs inline no-lang" v-pre>&lt;style&gt;</code> tag.</p>
-    <style>
-      .bigger-text {
-        font-size: 10em;
+    <script src defer type="application/javascript" style-bypass-vue-compilation>
+      .bigger - text {
```

Comment:
👀 

sorry, just noticed this as well. Could you investigate it? @wxwxwxwx9

Reply:
Hmm, I noticed this as well but I figured it shouldn't be a cause of concern, since we are not running the script and the conversion back to <style> tag doesn't seem to have any problem. Do you think it may cause a problem?

Reply:
how about just js-beautify's `content_unformatted` option? though this means blanket ignoring all script tags which should be ok

Reply:
That should work fine as well, but do you think we should put a notice for it in `https://markbind.org/userGuide/siteJsonFile.html#disablehtmlbeautify` accordingly? That `<script>` tags are not beautified by default.

Reply:
let's keep it in mind for now.

beautifying might not even make sense after ssr with the "gibberish" output (current purpose is to allow the author to figure out markdown nuances / etc. causing unexpected outputs)